docs/conceptual/runtime/frame.md

```diff
@@ -44,47 +44,47 @@ It also has a number of system critical storage items, such as:
 Finally, it defines a number of low level functions which can access your blockchain storage, verify
 the origin of an extrinsic, and more.
 
-### Executive Module
+### Executive Pallet
```

Comment:
Commenting here as the above (line 36) is out of comment zone.

"System critical" should have a hyphen. "a number of system-critical storage items"